### DISCUSSION TITLE
Multiple keys

### DISCUSSION THREAD

**Discussion by Hyppias** [SeriousM/WPFLocalizationExtension#318] [Ideas]
Wouldn't it be nice if We could localize in XAML like this:

Content="{lex:Loc We Need To Localize}"
or
Content="{lex:Loc Car}" for one single car 
Content="{lex:Loc Car s}"  for plural

where all keys are in Resources separately. one for We, one for Need, etc.

**Comment by konne** [comment]
Thank you for your suggestion. Feel free to provide a PR with a possible solution. I don't see at the moment the advantage and you can already use [SmartFormat.NET](https://github.com/axuno/SmartFormat/wiki) if you have added this as a dependency to your project.
